lib/madmin/resource.rb

```diff
@@ -74,7 +74,7 @@ def permitted_params
       end
 
       def display_name(record)
-        "#{record.class} ##{record.id}"
+        record.respond_to?(:display_name) ? record.display_name : "#{record.class} ##{record.id}"
```

Comment:
I think we can leave this as is. I'd rather the admin was separated from the models and we can override the `display_name` method in the resource model.

Instead, let's add a commented out example to the resource template: https://github.com/excid3/madmin/blob/master/lib/generators/madmin/resource/templates/resource.rb.tt

Something like this:

```ruby
class <%= class_name %>Resource < Madmin::Resource
  # ... 

  # Uncomment this to customize the display name of records in the admin area.
  # def display_name(record)
  #   record.name
  # end
end
```

Reply:
Yes, this is a good idea :+1: That would also "kind of" document this feature.